app/src/main/java/com/expidev/gcmapp/json/AssignmentsJsonParser.java

```diff
@@ -31,6 +31,12 @@
 
                 assignment.setId(assignmentJson.getString("id"));
                 assignment.setTeamRole(assignmentJson.getString("team_role"));
+                
+                JSONObject location = assignmentJson.getJSONObject("location");
```

Comment:
FYI... The location and zoom properties are meant to determine the initial map view of a ministry. This data does not exist yet (but I have manually set it for Guatemala). The web app will provide a way for the initial map view of a ministry to be set. I may try to do some automatic stuff based on the churches assigned to them (i have their locations), or their iso code. 

But location has nothing to do with if you are a leader or not. It is if location information exists for that ministry.